src/components/ProjectSetUp/ProjectSetUp.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from "react";
+import { ProjectInfoData } from "../../types/interfaces";
+import Grid from "@mui/material/Grid2";
+import ProjectForm from "./ProjectForm";
+import { Typography } from "@mui/material";
+
+const ProjectSetUp: React.FC = () => {
+  const [projectData, setProjectData] = useState<ProjectInfoData>({
+    projectName: "",
+    projectDescription: "",
+  });
+
+  const [projectFiles, setProjectFiles] = useState<FileList | null>(null);
+
+  useEffect(() => {
```

Comment:
We have a hooks folder, right? I don't know if it would be good if you could take the hooks you have, like this one, to that folder so that it has its functionality inside and the code is cleaner. I don't know if you agree with that, or maybe you could tell me why you wouldn't change it?

![image](https://github.com/user-attachments/assets/f853e2f8-8723-4e9d-a602-837f6625449d)

Reply:
This is not a custom hook, it is an already existing react hook.